### DISCUSSION TITLE
create a way to restore backups in the /storage directory

### DISCUSSION THREAD

**Discussion by mohammadmrd74** [verdaccio/verdaccio#2476] [Ideas]
My packages are gone and I'd just copied my backup packages in the `/storage/data` directory but verdaccio doesnt show them and It shows only packages with `npm publish` command. How can I see the backup packages that I'd copied? Thanks in advance.

**Comment by juanpicado** [comment]
hi @mohammadmrd74 you would need to provide detailed description of your problem, unique thing I can figure out is you are sort using docker.

**Comment by mohammadmrd74** [comment]
I'm using docker for installing verdaccio. After publishing all my packages through the `npm publish` command, I had copied all files in the `/storage/data` directory (which holds all the published packages) to my backup directory. I had installed verdaccio again after my data were gone (for another reason). I would like to restore the packages using my backup directory. I thought coping all that data to the `storage/data/` will work but unfortunately it doesn't.  Is there a way to restore my packages?

**Reply by juanpicado** [reply]
@mohammadmrd74 
1. Are you mapping volumes?
2. If 1 is yes. Do you have the right permissions? see https://github.com/verdaccio/verdaccio/discussions/2282#discussioncomment-882215

And please, @mohammadmrd74  I can help you better if you provide examples, without examples is really really hard to figure it out.
